bundle/compliance/main.rego

```diff
@@ -37,3 +43,9 @@ findings = f {
 }
 
 metadata = common.metadata
+
+# reports unknown subtype
+errors contains error if {
+	not input.subType in gcptypes.subtypes
+	error := sprintf("unknown subtype: '%s'", [input.subType])
+}
```

Comment:
when a subtype is not in the `subtypes` set, this error comes up in the OPA result. meant to indicate developers they're using an unknown subtype

<img width="534" alt="Screenshot 2023-08-01 at 21 37 10" src="https://github.com/elastic/csp-security-policies/assets/20814186/6a99c07b-c1fd-468a-a9b8-23e391194bed">